autoscheduler/user_sessions/views.py

```diff
@@ -62,3 +63,14 @@ def get_saved_courses(request):
         if courses:
             response = courses
     return Response(response)
+
+@api_view(['GET'])
+def get_full_name(request):
+    """ View that retrieves the first and last name seperated by a space for the user
+    of the current session """
```

Comment:
```suggestion
    """ View that retrieves the first and last name separated by a space for the user
          of the current session
    """
```

Note you also misspelled "separated"